admin/common_auth/views.py

```diff
@@ -0,0 +1,60 @@
+from django.http import HttpResponse
+from django.contrib import messages
+from django.contrib.auth import authenticate, logout as logout_user, login as auth_login, views
+from django.contrib.auth.decorators import login_required, user_passes_test
+from django.contrib.auth.models import User
+from django.shortcuts import render, redirect
+
+from forms import RegistrationForm, LoginForm
+from models import AdminUser
+
+#@login_required
```

Comment:
Why aren't we using `@login_required` here? It looks like it has a pretty flexible setup if we want to use it: https://docs.djangoproject.com/en/1.8/topics/auth/default/#the-login-required-decorator